src/applications/bmqstoragetool/m_bmqstoragetool_searchresult.h

```diff
@@ -369,8 +385,8 @@ class SearchResultTimestampDecorator : public SearchResultDecorator {
 
     // ACCESSORS
 
-    bool stop(bsls::Types::Uint64 timestamp) const;
-    // Return 'true' if the specified 'timestamp' is greated than
+    bool stop(const bsls::Types::Uint64 timestamp) const;
+    // Return 'true' if the specified 'timestamp' is greater than
```

Comment:
I see you're keeping the existing doc format here, which makes sense, but let's take this opportunity to convert this into Doxygen format---probably should have been from the beginning.